jlm/rvsdg/theta.hpp

```diff
@@ -29,68 +31,45 @@ class ThetaOperation final : public StructuralOperation
 
 class ThetaInput;
 class ThetaOutput;
+class ThetaResult;
```

Comment:
You just removed these classes below. These forward declarations can be removed as well, no?